codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/ServerServiceGeneratorV2.kt

```diff
@@ -178,18 +178,20 @@ class ServerServiceGeneratorV2(
 
     private fun buildMethod(): Writable = writable {
         val missingOperationsVariableName = "missing_operation_names"
-        val expectMessageVariableName = "unexpected_error_msg"
+        val expectMessage = "this should never panic since we are supposed to check beforehand that a handler has been registered for this operation; please file a bug report under https://github.com/awslabs/smithy-rs/issues"
```

Comment:
We used a variable there to avoid having multiple copies of the same string in the final binary artefact 🤔

Reply:
I see, I'd be surprised if it didn't deduplicate them. But I'll put a conditional `#[allow(unused_variables)]` instead.